apps/src/applab/dataBrowser/AddColumnButton.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * @overview Component for adding a new column to the specified table.
+ */
+
+import Radium from 'radium';
+import React from 'react';
+
+import * as dataStyles from './dataStyles';
+import { connect } from 'react-redux';
+
+const containerStyle = {
+  float: 'right',
+  paddingTop: 10,
+  paddingLeft: 10,
+  paddingBottom: 10,
+  paddingRight: 0
+};
+
+const AddColumnButton = React.createClass({
+  propTypes: {
+    addColumn: React.PropTypes.func.isRequired
+  },
+
+  handleAdd() {
+    this.props.addColumn();
+  },
+
+  render() {
+    return (
+      <div style={containerStyle}>
+        <button className="btn" onClick={this.handleAdd} style={dataStyles.button}>
+          Add column
+        </button>
+      </div>
+    );
+  }
+});
+export default Radium(AddColumnButton);
```

Comment:
Do we expect this add button to change much?  If not, I think there's a more idiomatic way to define it. You're not actually using `Radium` or `redux`, the whole thing is stateless, etc.  Any reason it shouldn't just be expressed this way?

```jsx
import React from 'react';
import {button as buttonStyle} from './dataStyles';

const containerStyle = {
  float: 'right',
  paddingTop: 10,
  paddingLeft: 10,
  paddingBottom: 10,
  paddingRight: 0
};

export default function AddColumnButton(props) {
  return (
    <div style={containerStyle}>
      <button className="btn" onClick={props.addColumn} style={dataStyles.button}>
        Add column
      </button>
    </div>
  );
}

AddColumnButton.propTypes = {
  addColumn: React.PropTypes.func.isRequired
};
```